pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerInstanceToggleTest.java

```diff
@@ -104,9 +107,21 @@ public void testInstanceToggle()
             .getPartitionSet().size(), 0);
   }
 
+  private void toggleInstanceState(String instanceName, String state) {
+    TestUtils.waitForCondition(aVoid -> {
+      try {
+        sendPostRequest(_controllerRequestURLBuilder.forInstanceState(instanceName), state);
+      } catch (IOException ioe) {
+        // receive non-200 status code
+        return false;
+      }
+      return true;
+    }, TIMEOUT_MS, "Failed to toggle instance state: \'" + state + "\' for instance: " + instanceName);
```

Comment:
Why are you logging `\` lol

Reply:
I'm logging `'` instead of `\`. Here's the simple program:
```
  public static void main(String[] args) {
    System.out.println("\'");
    System.out.println("'");
  }
```
Output:
```
'
'
```

Reply:
Good to learn. While that is for escaping quote purpose, which is not needed here.

Reply:
I found it useful to quote user input so that we can identify cases where the user may have input a bunch of spaces or null string. In that case, the log will show **Failed to toggle instance state: ' '** as opposed to **Failed to toggle instance state: **. 
But yes, we should not need the back-slash

Reply:
Back-slash removed.